sdk/configuration/__tests__/index.test.js

```diff
@@ -152,4 +156,50 @@ describe('configuration module', () => {
       scope: '',
     });
   });
+  it('eraseCode works correctly', () => {
+    const parameters7 = {
+      redirectUri: 'as',
+      clientId: 'as',
+      clientSecret: 'as',
+      code: 'as',
+      accessToken: 'as',
+      refreshToken: 'as',
+      tokenType: 'as',
+      expiresIn: 'as',
+      idToken: 'as',
+      postLogoutRedirectUri: 'as',
+      state: 'as',
+      scope: 'as',
+    };
```

Comment:
Agregar un enter con el test anterior. También cambiar los `as` por los nombres de las variables o algo un poco más formal.

Reply:
Agregar enter antes del `it` por favor para que quede separado del test anterior.